src/libraries_io_scraper/app/app.py

```diff
@@ -43,12 +42,12 @@ def py(dependency_file: str, output: str, template: str):
         output=output,
         template=template,
         parser=python.parse_dependencies_file,
-        platform="pypi",
+        platform="conda",
```

Comment:
I wonder if we need to do something about this too.

The previous philosophy was to get information from pypi _and_ conda and choose whichever results were better (because they're fundamentally looking at the exact same stuff, just via different ecosystems). Watcha think?

Reply:
added a pass-in-parameter-thing-i-forget-the-word